clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -1750,7 +1768,7 @@ public void testLongPollWaitIsLimited() {
     }
 
     /**
-     * Tests {@link AsyncKafkaConsumer#processBackgroundEvents(Future, Timer) processBackgroundEvents}
+     * Tests {@link AsyncKafkaConsumer#processBackgroundEvents(Future, Timer, java.util.function.Predicate) processBackgroundEvents}
```

Comment:
nit: we could simplify all java docs that now reference Predicate with the package name if we import it